tensorrt_llm/executor/proxy.py

```diff
@@ -60,6 +62,12 @@ def __init__(
         self.workers_started = False
         self.worker_cls = worker_cls
 
+        set_prometheus_multiproc_dir()
```

Comment:
If this is necessary, how about moving this to trtllm-serve as prometheus is for service deployment? For core API level, it may not be deployed in a cluster.